.github/workflows/openqa.yml

```diff
@@ -3,12 +3,17 @@ name: openQA - GitHub action example
 # yamllint disable-line rule:truthy
 on:
   # triggering on PRs is disabled here in favor of using the webhook-based approach
-  #pull_request:
+  # note: Use `pull_request_target` and not `pull_request` to enabled it. Only then
+  #       the workflow runs within the context of the main repository. This is
+  #       necessary to allow accessing the secrets for API access.
+  #pull_request_target:
```

Comment:
So "adapt" means switch off? I'm not clear what you want to achieve here.

Reply:
It has already been switched off before. This PR is not changing that. I'm just changing the already disabled example code here so it is covering the use case of creating a PR from a fork repository.